test/powershell/Modules/Microsoft.PowerShell.Utility/Measure-Object.Tests.ps1

```diff
@@ -189,6 +189,29 @@ Describe "Measure-Object" -Tags "CI" {
         }
     }
 
+    Context "Empty folder tests" {
+        BeforeAll {
+            $repoPath = Join-Path -Path $TESTDRIVE -ChildPath "my_folder"
+            $testEmptyFolder = New-Item $repoPath -ItemType "directory"
+            $propertyName = "Length"
+        }
+
+        It "Should not throw an invalid property error when not in StrictMode" {
+            { Set-StrictMode -off; Get-Item $testEmptyFolder | Measure-Object $propertyName -ErrorAction Stop -sum } | Should -Not -Throw
+        }
+    
+        It "Should throw an invalid property error when in StrictMode" {
+            Set-StrictMode -version 3.0
+            Get-Item $testEmptyFolder | Measure-Object -ErrorVariable Err $propertyName -sum -ErrorAction SilentlyContinue
+            $Err.count | Should -Be 1
+            $Err[0].FullyQualifiedErrorId | Should -BeExactly "GenericMeasurePropertyNotFound,Microsoft.PowerShell.Commands.MeasureObjectCommand"
+        }
+
+        AfterAll {
```

Comment:
This should be after BeforeAll.

Reply:
Up!

Reply:
I am a little confused by this suggestion.

Reply:
In all our test files are formatted so
https://github.com/PowerShell/PowerShell/blob/7bebf442affc1958387fe5f8537e9ad72e2fd02f/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Service.Tests.ps1#L3-L15

Reply:
Ah, I see, thank you